internal/generator/fft/template/domain.go.tmpl

```diff
@@ -45,37 +45,41 @@ type Domain struct {
 	cosetTableInv         []fr.Element
 }
 
-
-
-// NewDomain returns a subgroup with a power of 2 cardinality
-// cardinality >= m
-// shift: when specified, it's the element by which the set of root of unity is shifted.
-func NewDomain(m uint64, opts ...DomainOption) *Domain {
-	opt := domainOptions(opts...)
-	domain := &Domain{}
-	x := ecc.NextPowerOfTwo(m)
-	domain.Cardinality = uint64(x)
-
+// GetGenFrStar returns a generator of Z/rZ*
+func GetGenFrStar() fr.Element {
```

Comment:
And I think the method name could be different. Imo we usually do not use the `Get...` prefix for returning something. Maybe something a la `SubgroupGenerator` or `GeneratorSubgroup` etc considering we already have method `Generator`.